docs/pages/database-access/guides/elastic.mdx

```diff
@@ -0,0 +1,315 @@
+connect to.
+
+For example, if your Elasticsearch server is accessible at `elastic.example.com`,
+run:
+
+```code
+$ tctl auth sign --format=elastic --host=elastic.example.com --out=elastic --ttl=2160h
+Database credentials have been written to elastic.key, elastic.crt, elastic.cas.
+```
+
+(!docs/pages/includes/database-access/ttl-note.mdx!)
+
+The command will create three files:
+
+- `elastic.cas` with Teleport's certificate authority
+- `elastic.key` with a generated private key
+- `elastic.crt` with a generated host certificate
+
+(!docs/pages/includes/database-access/rotation-note.mdx!)
+
+Use the generated secrets to enable mutual TLS in your `elasticsearch.yml` configuration
+file:
+
+```yml
+xpack.security.http.ssl:
+  certificate_authorities: /path/to/elastic.cas
+  certificate: /path/to/elastic.crt
+  key: /path/to/elastic.key
+  enabled: true
+  client_authentication: required
+  verification_mode: certificate
+
+xpack.security.authc.realms.pki.pki1:
+  order: 1
+  enabled: true
+  certificate_authorities: /path/to/elastic.cas
+```
+
+Once mutual TLS has been enabled, you will no longer be able to connect to the cluster without
+providing a valid client certificate. You can set `xpack.security.http.ssl.client_authentication`
+to `optional` to allow connections from clients that do not present a certificate, using other
+methods like username and password.
+
+## Step 5/5. Connect
+
+Log into your Teleport cluster and see available databases:
+
+<Tabs>
+  <TabItem label="Self-Hosted">
+    ```code
+    $ tsh login --proxy=teleport.example.com --user=alice
+    $ tsh db ls
+    Name                        Description Allowed Users Labels  Connect
+    --------------------------- ----------- ------------- ------- ------------------------
+    > myelastic (user: elastic)             [*]           env=dev tsh db connect myelastic
+    ```
+  </TabItem>
+  <TabItem label="Teleport Cloud">
+    ```code
+    $ tsh login --proxy=mytennant.teleport.sh --user=alice
+    $ tsh db ls
+    Name                        Description Allowed Users Labels  Connect
+    --------------------------- ----------- ------------- ------- ------------------------
+    > myelastic (user: elastic)             [*]           env=dev tsh db connect myelastic
+    ```
+  </TabItem>
+</Tabs>
+
+To connect to a particular database instance:
+
+```code
+$ tsh db connect myelastic --db-user=alice
+```
+
+To log out of the database and remove credentials:
+
+```code
+#Remove credentials for a particular database instance.
+$ tsh db logout myelastic
+#Remove credentials for all database instances.
+$ tsh db logout
+```
+
+## Tunneled connection example
+
+We can create a tunneled connection to Elasticsearch to use with GUI applications like Elasticvue:
+
+```code
+$ tsh proxy db myelastic --db-user=alice --tunnel
+Started authenticated tunnel for the  database "myelastic" in cluster "teleport.example.com" on 127.0.0.1:53657.
```

Comment:
```suggestion
Started authenticated tunnel for the Elasticsearch database "myelastic" in cluster "teleport.example.com" on 127.0.0.1:53657.
```

`tsh` in 11.0.0-rc1 fixes the missing "Elasticsearch", not sure why it was missing in the first place.

Reply:
@Tener can you link the PR? I'd like to see if it was or will be backported to 10.

Reply:
I don't have the PR, I just noticed that 11.0.0-alpha1 was missing "Elasticsearch" here while the latest version did not. V10 should behave the same.